docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '2'
+
+services:
+    db:
```

Comment:
`user: "1000:50"` is needed to create a DB on OSX

Reply:
done.